definitions/install.yaml

```diff
@@ -26,48 +26,36 @@ spec:
       name: Background
       type: string
     - jsonPath: .spec.validationFailureAction
-      name: Validatoin Failure Action
+      name: Validation Failure Action
       type: string
     name: v1
     schema:
       openAPIV3Schema:
-        description: ClusterPolicy declares validation, mutation, and generation behaviors
-          for matching resources.
+        description: ClusterPolicy declares validation, mutation, and generation behaviors for matching resources.
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
-              of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
I think we already have these changes in the latest install.yaml, maybe yours are out-of-sync?
Anyway, it should generate the same manifest from the make command.